### PR TITLE
Issue templates

### PR DESCRIPTION
**Pull request by cjdb**
This (meta-)commit makes it possible for users of Compiler Explorer to indicate if their issue is a feature request or a bug (or something else).

### CODE REVIEW

**Comment by partouf** [comment]
I would love it if the issue template would contain something about making a shared link and pasting that, because that's usually more useful than a screenshot or reproduce step description

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG]"
+labels: bug
+assignees: cjdb
```
Comment (RabsRincon) [inline .github/ISSUE_TEMPLATE/bug_report.md]: I dont think you want to be assigned to every bug request :)

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
```
Comment (RabsRincon) [inline .github/ISSUE_TEMPLATE/feature_request.md]: There is a request label available

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (cjdb) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Wow, good catch, you're absolutely right 😉 
Whom should I change this to (anyone at all?)

**Comment by RabsRincon** [comment]
I think you can leave empty, as we already get notified in real time with any new issues :)
Thanks for taking the time to improve the user experience of the CE users :)

**Comment by RabsRincon** [comment]
I'll merge this now. Sorry that it took so long to get addressed!

**Pull request merged**